output/json.go

```diff
@@ -24,48 +25,58 @@ type JSONReport struct {
 	Behaviors map[string]*JSONBehaviorReport `json:"behaviors"`
 }
 
-var JSON ReporterFunc = func(config *plan.Config, tests <-chan execute.TestResponse) Summary {
-	summary := Summary{}
-	report := JSONReport{
-		Behaviors: make(map[string]*JSONBehaviorReport),
+type JSON struct {
+	s Summary
```

Comment:
You're not incrementing the counts in the json reporter... this means summary will be empty